astataudit

```diff
@@ -100,7 +100,7 @@ _add_column(){
     SECTION="${2}"
     KEY="${3}"
     VALUE="${4}"
-    if [[ "${FULL_DATA}" = "n" && "${IF_FULL}" = "n" ]] ; then
+    if [[ "${IF_FULL}" = "n" || ( "${FULL_DATA}" = "y" && "${IF_FULL}" = "y" ) ]] ; then
```

Comment:
I suggest you use here as well `==` rather than `=`, as done at lines 327 and 333 (or the contrary).